src/pages/admin/Admin.tsx

```diff
@@ -124,7 +134,88 @@ export const Admin = new Elysia({
         match_id: t.Number(),
       }),
     },
-  );
+  )
+  .post(
```

Comment:
This could be a put no?

Reply:
We are using post method in the form, just like when editing a match, and it won't work with put I'm afraid.